src/vorta/views/misc_tab.py

```diff
@@ -109,6 +116,37 @@ def populate(self):
 
         self.set_icons()
 
+    def on_setting_update(self, sender, instance: SettingsModel, created=False):
+        """
+        Handle a update of the settings db.
+        Non-PyQt slot for peewee's `playhouse.signals` api.
+        It calls `update_checkbox`.
+        Parameters
+        ----------
+        sender : Type[SettingsModel]
+            table sending model
+        instance : SettingsModel
+            The model instance (row) saved.
+        created : bool, optional
+            Whether it was newly created, by default False
+        """
+        if not created and instance.type == 'checkbox':
+            self.update_checkbox(instance.key, instance.value)
+
+    def update_checkbox(self, key, value):
+        """
+        Update the checkbox for a setting with a given key.
+        Parameters
+        ----------
+        key : str
+            The key of the setting to update.
+        value : bool
+            The value to set the checkbox to.
+        """
+        checkbox = self.settings_checkboxes[key]
```

Comment:
```suggestion
        checkbox = self.settings_checkboxes.get(key)
```

This will hopefully fix the tests.

Reply:
Hm, this still fails the tests locally